src/csync/csync_exclude.cpp

```diff
@@ -361,29 +361,34 @@ CSYNC_EXCLUDE_TYPE ExcludedFiles::traversalPatternMatch(const char *path, ItemTy
 
     QRegularExpressionMatch m;
     if (filetype == ItemTypeDirectory) {
-        m = _bnameActivationRegexDir.match(bnameStr);
+        m = _bnameTraversalRegexDir.match(bnameStr);
     } else {
-        m = _bnameActivationRegexFile.match(bnameStr);
+        m = _bnameTraversalRegexFile.match(bnameStr);
     }
     if (!m.hasMatch())
-        return match;
-
-    // Now run the full match
+        return CSYNC_NOT_EXCLUDED;
+    if (!m.captured(1).isEmpty()) {
```

Comment:
Just thinking:  Wouldn't it make the code more readable if we used named captures?  Or would that be slower?

Reply:
Good point, I'll test that!